src/datepicker/datepicker-navigation.ts

```diff
@@ -1,55 +1,99 @@
 import {Component, Input, Output, EventEmitter, ChangeDetectionStrategy} from '@angular/core';
-import {NavigationEvent} from './datepicker-view-model';
+import {NavigationEvent, MonthViewModel} from './datepicker-view-model';
 import {NgbDate} from './ngb-date';
 import {NgbDatepickerI18n} from './datepicker-i18n';
 import {NgbCalendar} from './ngb-calendar';
 
 @Component({
   selector: 'ngb-datepicker-navigation',
   changeDetection: ChangeDetectionStrategy.OnPush,
-  host: {'class': 'd-flex justify-content-between', '[class.collapsed]': '!showSelect'},
+  host: {'class': 'd-flex'},
   styles: [`
     :host {
       height: 2rem;
       line-height: 1.85rem;
     }
-    :host.collapsed {
-      margin-bottom: -2rem;
-    }
     .ngb-dp-navigation-chevron::before {
       border-style: solid;
       border-width: 0.2em 0.2em 0 0;
       content: '';
       display: inline-block;
+      width: 0.75em;
       height: 0.75em;
       transform: rotate(-135deg);
       -webkit-transform: rotate(-135deg);
       -ms-transform: rotate(-135deg);
-      width: 0.75em;
-      margin: 0 0 0 0.5rem;
     }
-    .ngb-dp-navigation-chevron.right:before {
+    .ngb-dp-arrow-btn.right {
+      float: right;
+    }
+    .right .ngb-dp-navigation-chevron:before {
       -webkit-transform: rotate(45deg);
       -ms-transform: rotate(45deg);
       transform: rotate(45deg);
-      margin: 0 0.5rem 0 0;
+    }
+    .btn-link {
```

Comment:
I don't think we need to override `.btn-link` at all, let's stick with default bootstrap styling
Please see #1948, where it was removed previously

Reply:
I removed the override of btn-link